website/cue/reference/components/sinks/gcp_stackdriver_metrics.cue

```diff
@@ -139,6 +139,12 @@ components: sinks: gcp_stackdriver_metrics: {
 	}
 
 	how_it_works: {
+		duplicate_tags: {
+			title: "Duplicate tags"
+			body: """
+					Duplicate tag names cannot be sent to GCP. Vector will only send the last tag specified.
+				"""
```

Comment:
Some recommendations to try to clarify the intent:
```suggestion
		duplicate_tag_names: {
			title: "Duplicate tag names"
			body: """
					Multiple tags with the same name cannot be sent to GCP. Vector will only send the last value for each tag name.
				"""
```

Reply:
Yes, that's much better.